JointMeasurements/Tests.qs

```diff
@@ -22,6 +22,9 @@ namespace Quantum.Kata.JointMeasurements {
     operation DistinguishStates_MultiQubit (Nqubit : Int, Nstate : Int, statePrep : ((Qubit[], Int, Double) => Unit is Adj), testImpl : (Qubit[] => Int), preserveState : Bool) : Unit {
         let nTotal = 100;
         mutable nOk = 0;
+        mutable misclassifications = new String[Nstate];
```

Comment:
The CI build fails because it's claiming that `Nstate` identifier doesn't exist. This is a bit weird, since it definitely does exist... Maybe with the next changes this will go away?